### PR TITLE
Refactoring/gloabal

### PR DESCRIPTION
**Pull request by GarifullinMarsel**

### CODE REVIEW

**Comment by pokedim13** [comment]
I don't see where edits are needed anymore, great job!

**Comment by pokedim13** [comment]
And give the neural readme) Or fix it, I'll fix it myself if something happens later

File: wombo/base_models/styles.py [hunk 1]
```diff
@@ -1,80 +1,60 @@
...
-                    def sync_func(text: str):
-                        return self.dream.generate(text=text, style=list(styles_list.keys())[list(styles_list.values()).index(param.lower())])
-                    return sync_func
-                elif isinstance(self.dream, AsyncDream):
-                    async def async_func(text: str):
-                        return self.dream.generate(text=text, style=list(styles_list.keys())[list(styles_list.values()).index(param.lower())])
-                    return async_func
-                else:
-                    return None
+class Style(Enum):
```
[the remark this answers is not in the document]
Comment (GarifullinMarsel) [reply wombo/base_models/styles.py:4]: can't implement at the moment

File: wombo/base_models/base_dream.py [hunk 2]
```diff
@@ -1,18 +1,22 @@
...
 
-class BaseDream:
+
+class BaseDream(ABC):
     """
     Base class for all dreams (sync / async).
     """
 
-    client: BaseClient
+    client: Union[Client, AsyncClient]
```
[the remark this answers is not in the document]
Comment (GarifullinMarsel) [reply wombo/base_models/base_dream.py:16]: added

File: wombo/base_models/styles.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (pokedim13) [reply wombo/base_models/styles.py:4]: Url for get styles 
https://dream.ai/_next/data/ddBu0LvpgvZcniN77Wr0h/create.json
trans: Ссылка для получения стилей

File: wombo/api/async_dream.py [hunk 3]
```diff
@@ -17,15 +20,17 @@ def __init__(self, out_msg: str = "") -> None:
         self.out_msg: str = out_msg
```
[the remark this answers is not in the document]
Comment (GarifullinMarsel) [reply wombo/api/async_dream.py:21]: added

**Pull request closed without merging**